File: Java/Car.java

public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passegenger;

    public Car(String license, Account driver){
        this.license=license;
        this.driver = driver;
    }

    public void imprimeDatos(){
        System.out.println("License: "+license+" driver: "+driver.name);
    }
}

File: PHP/Car.php
<?php
    require_once(Account.php);
    class Car(){
        public $id = integer;
        public $license = string;
        public $driver = string;
        public $passengers = integer;
        
        public function __construct($license, $driver){
            $this->license = $license;
            $this->dirver = $driver;
        }

        public function printDataCar(){
            echo "Licencia:  $this->$license Driver:".$this->driver->name;
        }
    }
    
?>